.github/workflows/ci.yml

```diff
@@ -19,6 +19,36 @@ jobs:
     - name: module api test
       run: ./runtest-moduleapi --verbose
 
+  test-sanitizer-address:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: make
+        run: make SANITIZER=address REDIS_CFLAGS='-DREDIS_TEST'
+      - name: testprep
+        run: sudo apt-get install tcl8.6 tclx -y
+      - name: test
+        run: ./runtest --sanitizer --verbose --tags -slow
+      - name: module api test
+        run: ./runtest-moduleapi --sanitizer --verbose
+      - name: unittest
+        run: ./src/redis-server test all
```

Comment:
is there any reason to keep the other `test-ubuntu-latest` job?
i was thinking we can just re-purpose the one "per PR" test job we have to do both testing and address (leak) sanitization.
is there any downside in that?

Reply:
It's up to you. Sanitizers instrument code and may hide some bugs due to slower execution, different code generation etc. Ofcourse you can rely on daily CI for that, we still have default build and tests there.

Reply:
ok, we can keep both and maybe drop one later... 
i guess i'm just trying to save Microsoft some cache on CPU time (or care about global warming)